### PR TITLE
Added italian language

### PR DESCRIPTION
**Pull request by phaberest**
Nothing too fancy, I just added the italian translation 😃

### CODE REVIEW

**Comment by charliekassel** [comment]
Hey thanks for your contribution @phaberest !

File: README.md [hunk 1]
```diff
@@ -99,3 +99,4 @@ Available languages
 | vi          | Vietnamese       |          |
 | zh          | Chinese          |          |
 | ja          | Japanese         |          |
+| it          | Italian          |          |
```
Comment (charliekassel) [inline README.md]: Can you move this up into the European block?

Comment (phaberest) [inline README.md]: Sure, give me a sec ;)

**Comment by phaberest** [comment]
I repositioned it alphabetically under fr in both files

**Comment by charliekassel** [comment]
Great thanks!

**Pull request merged**